src/App.js

```diff
@@ -219,6 +220,7 @@ class App extends React.Component {
 
   render() {
     const {
+      fatalError,
```

Comment:
I don't see a spot where we are settings this. Do we still need it with the global error trap?